src/Show.jl

```diff
@@ -67,6 +87,24 @@ function classtypes(t::Table)
 	size(t.table, 2)
 end
 
+"""
+    classes(t::Table)
```

Comment:
As above I'd just call this `nrclasses` as well

And eventually rename both to `n_classes`, and `classtypes` to `n_classtypes` -- but please not in this PR, this needs to be carefully coordinated.